tests/test_helper.tcl

```diff
@@ -370,7 +371,13 @@ proc read_from_test_client fd {
         set all_tests_count [llength $::all_tests]
         set running_tests_count [expr {[llength $::active_clients]-1}]
         set completed_tests_count [expr {$::next_test-$running_tests_count}]
-        puts "\[$completed_tests_count/$all_tests_count [colorstr yellow $status]\]: $data ($elapsed seconds)"
+        # Solo tests are excluded from the completed_tests_count. Ref: #valkey/issues/1017
```

Comment:
i dont think we need this comment, we have a git blame log so we can save the issue ref.